agent/lib/kontena/service_pods/stopper.rb

```diff
@@ -21,8 +21,22 @@ def initialize(service_id, instance_number)
       def perform
         service_container = get_container(self.service_id, self.instance_number)
         if service_container.running?
-          info "stopping service: #{service_container.name}"
+          info "stopping service: #{service_container.name_for_humans}"
+
+          log_service_pod_event(
+            self.service_id, self.instance_number,
+            "service:stop_instance", "stopping service instance #{service_container.name_for_humans}"
+          )
           service_container.stop('timeout' => 10)
+          log_service_pod_event(
+            self.service_id, self.instance_number,
+            "service:stop_instance", "service instance #{service_container.name_for_humans} stopped successfully"
+          )
+        else
+          log_service_pod_event(
+            self.service_id, self.instance_number,
+            "service:stop_instance", "service instance #{service_container.name_for_humans} is not running"
+          )
```

Comment:
This produces kinda empty-hanging event. There's no event that stop is requested (at least on agent side) so having an event saying "service instance 3 is not running" can be a bit confusing :)